sdk/identity/azure-identity/samples/CMakeLists.txt

```diff
@@ -7,6 +7,11 @@ project (azure-identity-samples LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
+add_executable(chained_token_credential_sample chained_token_credential.cpp)
+target_link_libraries(chained_token_credential_sample PRIVATE azure-identity)
+target_include_directories(chained_token_credential_sample PRIVATE .)
```

Comment:
I no longer remember, what does the `.` do here for `target_include_directories`?

Reply:
If I remember correctly, it allows you to include files from the same directory as the cpp file using the `<>` syntax.